unit_test/sample_key/readme.txt

```diff
@@ -294,3 +294,23 @@ openssl asn1parse -in Shorter1024B_end_responder.cert -out Shorter1024B_end_resp
 cat Shorter1024B_ca.cert.der Shorter1024B_end_requester.cert.der > Shorter1024B_bundle_requester.certchain.der
 cat Shorter1024B_ca.cert.der Shorter1024B_end_responder.cert.der > Shorter1024B_bundle_responder.certchain.der
 popd
+
+
+==== More cert_chain to gen ====
+
+NOTE: The bundle_requester.certchain1.der and bundle_requester.certchain.der have same leaf cert key.
+As same as bundle_responder.certchain1.der.
+
+The script is:
+openssl req -nodes -x509 -days 3650 -newkey rsa:4096 -keyout ca1.key -out ca1.cert -sha512 -subj "//CN=intel test RSA CA"
+
+openssl x509 -req -in inter.req -out inter1.cert -CA ca1.cert -CAkey ca1.key -sha512 -days 3650 -set_serial 1 -extensions v3_inter -extfile ../openssl.cnf
+openssl x509 -req -in end_requester.req -out end_requester1.cert -CA inter1.cert -CAkey inter.key -sha512  -days 3650 -set_serial 2 -extensions v3_end -extfile ../openssl.cnf

```

Comment:
Would you please remove end_requester1.cert and end_responder1.cert ?

we just need ca1.cert and inter1.cert.